mariadb.html.md.erb

```diff
@@ -47,4 +49,23 @@ Cloud Foundry: [Spring Music Example](https://github.com/cloudfoundry-samples/sp
 
 ## <a id='limitations'></a> Known Limitations
 
-The base of our HA MariaDB offering is built by a galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
+Our HA MariaDB offering is based on Galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
+
+Our Galera aaS is a shared Galera cluster. Each customer has their own database (`CREATE DATABSE`) with all privileges.
+
+```
+GRANT ALL PRIVILEGES ON `CF_DATABASE_NAME`.* TO 'USER'@'SUBNET'
```

Comment:
as discussed: Please change to:

Our Galera aaS is a shared Galera cluster. Each customer has their own database. Simplified, the service broker executes the following DDL statements:

```SQL
CREATE DATABSE CF_DATABASE_NAME
GRANT ALL PRIVILEGES ON `CF_DATABASE_NAME`.* TO 'USER'@'SUBNET'
```